config/fields/tags.php

```diff
@@ -27,6 +27,12 @@
         'icon' => function ($icon = 'tag') {
             return $icon;
         },
+        /**
+         * Set to `list` to show each tag as 100% width
+         */
+        'layout' => function ($layout = null) {
```

Comment:
No needed the type parameter for `data-layout` attribute? `string $layout = null`

Reply:
Oh Yes, type hinting totally makes sense there!

Reply:
Have added it